packages/server/src/healthcheck.ts

```diff
@@ -1,16 +1,30 @@
 import { MEDPLUM_VERSION } from '@medplum/core';
 import { Request, Response } from 'express';
 import { getDatabasePool } from './database';
+import { setGauge } from './otel/otel';
 import { getRedis } from './redis';
 
 export async function healthcheckHandler(_req: Request, res: Response): Promise<void> {
+  setGauge('medplum.db.idleConnections', getDatabasePool().idleCount);
+  setGauge('medplum.db.queriesAwaitingClient', getDatabasePool().waitingCount);
+
+  let startTime = Date.now();
+  const postgresOk = await testPostgres();
+  const dbRoundtripMs = Date.now() - startTime;
+  setGauge('medplum.db.roundtripMs', dbRoundtripMs);
```

Comment:
For clarity, we might want to name both of these roundtrip metrics to indicate that they're only for the healthcheck, and aren't measuring all queries — e.g. `medplum.db.healthcheckLatency`.

Additionally, we should figure out how to segment these metrics by host